src/templates/IFunctionTemplate.ts

```diff
@@ -19,6 +19,7 @@ export interface IFunctionTemplate {
     language: string;
     isHttpTrigger: boolean;
     isTimerTrigger: boolean;
+    isSqlBindingTrigger: boolean;
```

Comment:
I'd suggest `isSqlBindingTemplate` - we don't (currently) have trigger support so this is slightly misleading as it is.